<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>产品库存管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>查看</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="loginToolBar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>状态</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="loginAct"/>
   <addaction name="ReLogin"/>
  </widget>
  <widget class="QToolBar" name="naviBar">
   <property name="windowTitle">
    <string>导航</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="homeAction"/>
   <addaction name="stockAction"/>
   <addaction name="inAction"/>
   <addaction name="outAction"/>
   <addaction name="userAction"/>
   <addaction name="separator"/>
  </widget>
  <action name="loginAct">
   <property name="text">
    <string>已登录</string>
   </property>
  </action>
  <action name="ReLogin">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="homeAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/home.png</normaloff>:/images/home.png</iconset>
   </property>
   <property name="text">
    <string>主页</string>
   </property>
  </action>
  <action name="stockAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/file.png</normaloff>:/images/file.png</iconset>
   </property>
   <property name="text">
    <string>库存管理</string>
   </property>
  </action>
  <action name="userAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/user.png</normaloff>:/images/user.png</iconset>
   </property>
   <property name="text">
    <string>用户管理</string>
   </property>
  </action>
  <action name="inAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>入库管理</string>
   </property>
  </action>
  <action name="outAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>出库管理</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
